test/functional/services/find.js

```diff
@@ -109,16 +109,27 @@ export async function FindProvider({ getService }) {
 
     async allByLinkText(selector, timeout = defaultFindTimeout) {
       log.debug('find.allByLinkText: ' + selector);
-      return await this.allByCustom(async driver => wrapAll(await driver.findElements(By.linkText(selector)), timeout));
+      try {
+        await this.byLinkText(selector, timeout);
+        return wrapAll(await driver.findElements(By.linkText(selector)));
+      } catch (err) {
+        return [];
```

Comment:
Can we test the error here to make sure it's the specific type of error we want to replace with an empty array? There are other types of errors that we want to rethrow.

Reply:
Agree, we can. I will specify it explicitly